specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
I certainly do not really care how the DB data model gets refined from what is in this spec, so long as each persistent datatype is realized (without fidelity loss) and each relation (as encoded in all of the `*_table` types) is obtainable via query.  Some of that encoding is going to be undoubtedly driven by whatever awkward schema you end up reverse engineering from Dominion resources.